src/wiringx.c

```diff
@@ -103,6 +86,86 @@ static struct spi_t spi[2] = {
 	} while (0)
 #endif
 
+#ifndef __FreeBSD__
+/* SPI Bus Parameters */
+struct spi_t {
```

Comment:
Why aren't these static?

Reply:
because it wasn't static before.

Reply:
I think that was a mistake.

Reply:
wiringX/src/wiringx.c:103:1: warning: useless storage class specifier in empty declaration [enabled by default]

No it was by design